### ISSUE TITLE
at-fastmath cannot be used with op= operations

### ISSUE THREAD

**Issue by pao** [JuliaLang/julia#10544]
The equivalent statement without the `op=` works fine.

```
julia> a = [0.]; @fastmath a[1]+=2.
ERROR: syntax: "(1,)" is not a valid function argument name

julia> a = [0.]; @fastmath a[1]*=2.
ERROR: syntax: "(1,)" is not a valid function argument name

julia> a = [0.]; @fastmath a[1]=a[1] * 2.
0.0

julia> versioninfo()
Julia Version 0.4.0-dev+3678
Commit 6d0db53* (2015-03-05 20:20 UTC)
Platform Info:
  System: Linux (x86_64-linux-gnu)
  CPU: Intel(R) Core(TM) i7-3612QE CPU @ 2.10GHz
  WORD_SIZE: 64
  BLAS: libopenblas ()
  LAPACK: liblapack.so.3
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

**Comment by jakebolewski** [comment]
Closed by 09604d6ff32f169b5

**Comment by pao** [comment]
Thanks!
